pkgs/tools/typesetting/pulldown-cmark/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+}:
+
+let
+  owner = "raphlinus";
+  pname = "pulldown-cmark";
+  version = "0.9.2";
+in
+rustPlatform.buildRustPackage {
+  inherit pname version;
+
+  src = fetchFromGitHub {
+    inherit owner;
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-AAb+dSJ1oSRuvWu47VvzCeB6pQE6/+u69io2FsZoZHM=";
+  };
+
+  cargoHash = "sha256-oOgwZMmrzYBFH1MaE7nMa1SPCACnfqYY3ttOECsnsVY=";
+
+  meta = {
+    description = "A pull parser for CommonMark written in Rust";
+    homepage = "https://github.com/${owner}/${pname}";
```

Comment:
I prefer this to be a literal url for easier copying

Reply:
Clickable links are highly preferred

Reply:
I wasn't aware of this, I just assumed deduplication was preferred. Thanks! This is fixed now.